docs/notary-mysql.md

```diff
@@ -0,0 +1,23 @@
+<!--[metadata]>
++++
+title = "Notary MySQL"
+description = "Description of the Notary MySQL"
+keywords = ["docker, notary, notary-mysql"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Notary MySQL
+
+The Notary MySQL is one of the backends for [Notary Server](notary-server.md) and
+[Notary Signer](notary-signer.md).
+
+### Recommendation
+For security, especially in production. Before ever deployed one could create
```

Comment:
Maybe "For security, especially in production deployments, one should create users..."?